python/pylibcudf/CMakeLists.txt

```diff
@@ -53,7 +53,7 @@ if(FIND_CUDF_CPP)
     include(../../cpp/cmake/thirdparty/get_arrow.cmake)
   endif()
 
-  find_package(cudf "${RAPIDS_VERSION}" REQUIRED)
+  # find_package(cudf "${RAPIDS_VERSION}" REQUIRED)
```

Comment:
Same here.

I think this means look for the cudf C++ library files (not look for cudf python), so we'll need this uncommented as well.

Reply:
Okay, I was trying to get `build-cudf`/`build-cudf-python` working locally and they both fail at this line

Reply:
Here was the error

```
       Could not find a package configuration file provided by "cudf" (requested
        version 24.10.00) with any of the following names:
      
          cudfConfig.cmake
          cudf-config.cmake
      
        Add the installation prefix of "cudf" to CMAKE_PREFIX_PATH or set
        "cudf_DIR" to a directory containing one of the above files.  If "cudf"
        provides a separate development package or SDK, be sure it has been
        installed.
      
      
      -- Configuring incomplete, errors occurred!
```

Reply:
Hm, this works on my machine :\

Can you try rebuilding the devcontainer without cache?

Reply:
Okay I'll retry

Reply:
It worked!